haas/ext/switches/migrations/mock/b5b31d19257d_rename_mockswitch_table_for_flask_.py

```diff
@@ -7,17 +7,16 @@
 Create Date: 2016-03-22 05:11:19.585905
 
 """
+from alembic import op
```

Comment:
@zenhack @shwsun there are a few of these where the imports were not at the top, just want to make sure moving them won't cause an issue.  It passed the tests but just want to confirm, I can change them back if need be